### DISCUSSION TITLE
Add NumberOfTapsRequired property in TouchEffect

### DISCUSSION THREAD

**Discussion by Sergtek** [CommunityToolkit/Maui#293] [New Feature Discussions]
## Summary
My suggestion would be to add a new property called NumberOfTapsRequired to the TouchEffect, similar to the one that the [gesture recognizer](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/app-fundamentals/gestures/tap) has to establish how many clicks are necessary for the associated command to be executed.

## API Changes
e.g.
```
<StackLayout xct:TouchEffect.Command="{Binding SampleCommand}"
             xct:TouchEffect.CommandParameter="{Binding .}"
             xct:NumberOfTapsRequired="2">
```

## Intended Use Case
This functionality could be used in countless cases.

## Who Will Do The Work?
- [x]  Just putting this out there for someone to pick up

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
